README.md

```diff
@@ -36,6 +36,16 @@ These instructions will get you a copy of the project up and running on your loc
 
 _For more information regarding the use of Jekyll, please refer to the [Jekyll Step by Step Tutorial](https://jekyllrb.com/docs/step-by-step/01-setup/)._
 
+### Using Docker to run Jekyll
+
+Alternatively you can use Docker to test your changes locally.
+
+`
+docker run --rm --volume="$PWD:/srv/jekyll" -it -p 4000:4000 jekyll/jekyll:3 jekyll serve  
```

Comment:
maybe point to what PWD user must be just to enforce it?

Reply:
@spolti thanks, how about now?

Reply:
sounds good.